src/components/Charts/ModelChart.js

```diff
@@ -40,14 +41,15 @@ const ModelChart = ({
   height,
   condensed,
   projections,
-  currentIntervention,
-  lastUpdatedDate,
   forCompareModels, // true when used by CompareInterventions.js component.
   stateId,
   selectedCounty,
 }) => {
   // We use the inferred projection if supported, otherwise the worst case for the currently active intervention
   let projection = projections.primary;
+  const currentIntervention = STATE_TO_INTERVENTION[stateId];
```

Comment:
this is available as a field on projections ("stateIntevention") and should be exposed wit ha getter